static/app/utils/profiling/profile/chromeTraceProfile.tsx

```diff
@@ -510,24 +514,151 @@ export function collapseSamples(profile: ChromeTrace.CpuProfile): {
   return {samples, sampleTimes};
 }
 
-export function parseChromeTraceFormat(
+function shouldPlaceOnTopOfPreviousStack(_frame: ChromeTrace.CallFrame): boolean {
+  return false;
+}
+
+function lastOf<T extends any[]>(arr: T): T[number] {
+  return arr[arr.length - 1];
+}
```

Comment:
This already exists as a helper in https://github.com/getsentry/sentry/blob/master/static/app/utils.tsx#L77-L81